edith-step/src/UserChart.js

```diff
@@ -0,0 +1,219 @@
+import React, {useState, useEffect} from 'react';
+import {Line, Bar, Pie} from 'react-chartjs-2';
+import axios from "axios";
+
+const LineChart = () => {
+    
+  const [chartData, setChartData] = useState({});
+  const chart = () => {
+    axios
+      .get("/user-stats-servlet")
+      .then(response => {
+        console.log(response);
+        let weekDates = [];
+        let values = [];  
+        const weeklyAggregate = JSON.parse(response.data.weeklyAggregate)
+        weekDates = Object.keys(weeklyAggregate);
+        for (let i = 0; i < weekDates.length; i++) {
+          values.push(weeklyAggregate[weekDates[i]]);
+        }
+        setChartData({
+          labels: weekDates,
+          datasets: [
+            {
+              label: "Week Total",
+              data: values,
+              backgroundColor: ["rgba(75, 192, 192, 0.6)"],
+              borderWidth: 4
+            }
+          ]
+        });
+        console.log(weekDates, values);
+      })
+      .catch(err => {
+        console.log(err);
+      });
+    };
+
+  useEffect(() => {
+    chart();
+  }, []);
+
+  return (
+      <Line
+        data={chartData}
+        width={100}
+        height={100}
+        options={{
+          maintainAspectRatio: false,
+          title:{
+            display: true,
+            text: 'Weekly Aggregate (trailing total based on week date)',
+            fontSize: 20
+          }, 
+          legend:{
+            display: true,
+            position:'right'
+          },
+          scales: {
+            yAxes: [{
+              scaleLabel: {
+                display: true,
+                labelString: 'Dollars'
+              },
+              ticks: {
+                beginAtZero: true
+              }
+            }],
+            xAxes: [{
+              scaleLabel: {
+                display: true,
+                labelString: 'Week'
+              }
+            }],
+          }    
+        }}
+      />
+    );
+  }
+
+const BarGraph = () => {
+    
+  const [chartData, setChartData] = useState({});
+  const chart = () => {
+    axios
+      .get("/user-stats-servlet")
+      .then(response => {
+        console.log(response);
```

Comment:
can this print statement be deleted as well

Reply:
Yea